src/script/components/MessagesList/Message/ContentMessage/asset/ImageAsset/ImageAsset.tsx

```diff
@@ -50,7 +50,7 @@ export const ImageAsset = ({asset, message, onClick}: ImageAssetProps) => {
 
   const imageContainerStyle: CSSObject = {
     maxWidth: 'var(--conversation-message-asset-width)',
-    maxHeight: '80vh',
+    maxHeight: 'calc(100vh - 155px - 1rem)',
```

Comment:
unfortunately, we don't use css variable for the height of the relevant components, they are:
- Title bar: 45px
- Typing indicator: 40px
- Input Bar: 56px
- Image padding: 12px
- Classified bar: 1rem

153px + 1rem, rounded up to 155px

Reply:
This should appear as a comment in the code ;)